tools/util/include/cutlass/util/device_memory.h

```diff
@@ -140,12 +156,16 @@ class DeviceAllocation {
   /// Delete functor for CUDA device memory
   struct deleter {
     void operator()(T* ptr) {
+#ifdef CUTLASS_ENABLE_SYCL
+      syclcompat::free(reinterpret_cast<void*>(ptr));
```

Comment:
nullptr check here as well

Reply:
There is a nullptr check for this in the file https://github.com/codeplaysoftware/cutlass-fork/blob/8a456718f944bdb6497b114a336c5d8138579234/include/cutlass/platform/platform.h#L842